.github/workflows/pip-requirements.yml

```diff
@@ -44,13 +44,12 @@ jobs:
       - name: Setup environment
         working-directory: ncs
         run: |
-          pip3 install --user -U setuptools wheel pip virtualenv virtualenvwrapper
-          pip3 install -r nrf/scripts/requirements-base.txt
+          pip3 install --user -U setuptools wheel pip virtualenv virtualenvwrapper west
           west init -l nrf
           west update --narrow mcuboot zephyr
 
       - name: Execute script diff action
-        uses: nordicbuilder/action-script-diff@main
+        uses: nordicbuilder/action-script-diff@v0.1
```

Comment:
The version used in the 'main' branch points to 'v0.2'

Reply:
Changed it. Target should be to make minimal changes to make it safe. I don't think we need (maybe even cannot) do all the changes we have in main.